ViewLayer/OverlordWidget/OverlordWidget.cpp

```diff
@@ -0,0 +1,133 @@
+#include <QVBoxLayout>
+#include "OverlordWidget.h"
+#include "BatteryUi/I_BatteryUi.h"
+#include "ControlUi/I_ControlUi.h"
+#include "HomePageUi/I_HomePageUi.h"
+#include "MotorFaultUi/I_MotorFaultUi.h"
+#include "MotorUi/I_MotorUi.h"
+#include "MPPTUi/I_MpptUi.h"
+#include "TabUi/I_TabUi.h"
+OverlordWidget::OverlordWidget(I_BatteryUi& batteryUi
+                               , I_ControlUi& controlUi
+                               , I_HomePageUi& homepageUi
+                               , I_MotorFaultUi& motorFaultUi
+                               , I_MotorUi& motorUi
+                               , I_MpptUi& mpptUi
+                               , I_TabUi& tabUi
+                              )
```

Comment:
Only initialisation lists should be formatted like this. The arguments should be formatted like:
`OverlordWidget::OverlordWidget(I_BatteryUI& batteryUi,`
`                I_ControlUi& controlUi,`
`...etc`

See the way `DashboardDisplayView.cpp` has it formatted.